src/howler.core.js

```diff
@@ -303,6 +309,21 @@
       // then check if the audio actually played to determine if
       // audio has now been unlocked on iOS, Android, etc.
       var unlock = function() {
+        // Create a pool of activated HTML5 Audio objects that can
+        // be used for playing sounds without user interaction. HTML5
+        // Audio objects must be individually activated, as opposed
+        // to the WebAudio API which only needs a single activation.
+        // This must occur before WebAudio setup or the source.onended
+        // event will not fire.
+        Howler._mobileAudioPool = [];
```

Comment:
Should we call this `_html5AudioPool` or `_mobileHtml5AudioPool`? So that it's further differentiated from `_webAudio`?